tools/cli/commands/create.py

```diff
@@ -773,6 +816,7 @@ def run(args, gcloud_compute, gcloud_repos,
                 '--format=none',
                 '--boot-disk-size=20GB',
                 '--network', args.network_name,
+                '--subnet', args.subnet_name,
```

Comment:
It's important that this be conditional on the `--subnet-name` flag being provided.

For instance, if someone wants to create a Datalab instance connected to a legacy network, then they can't have any value for the `--subnet` flag.